libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Tests.hs

```diff
@@ -708,7 +709,7 @@ defaultWhitelist = ["Size specifications", "The SetSpec's are inconsistent", "Th
 --   constraints.
 prop_soundness' :: Bool -> [String] -> OrderInfo -> Property
 prop_soundness' strict whitelist info =
-  constraintProperty (Just 100000) strict whitelist info $ \preds _graph env ->
+  constraintProperty (Just 200000) strict whitelist info $ \preds _graph env ->
```

Comment:
The advise was to remove this argument and a call to `within` from `constraintProperty`. Also .2 seconds is too low. Let's use at least one second.
```suggestion
  within 1_000_000 $
  constraintProperty strict whitelist info $ \preds _graph env ->
```